### ISSUE TITLE
[BUG] "timeout=None" Not Working

### ISSUE THREAD

**Issue by Zai-Kun** [yifeikong/curl_cffi#142]
**To Reproduce**
```python
from curl_cffi.requests import AsyncSession
import asyncio

async def main():
    async with AsyncSession(impersonate="chrome110") as session:
        session.get(url="some url", timeout=None)

asyncio.run(main())
```
**Expected behavior**
It shouldn't time out, but it times out after 3 seconds.

**Versions**
 - OS: linux x64
 - curl_cffi version: 0.5.9

**Comment by Zai-Kun** [comment]
> Try setting timeout to session instead and see if it works.

It throws this error:
`
c.setopt(CurlOpt.TIMEOUT_MS, int(timeout * 1000))
TypeError: unsupported operand type(s) for *: 'NoneType' and 'int'
`

**Comment by rubmed90** [comment]
Set timeout to a very high a number then.

**Comment by Zai-Kun** [comment]
> Set timeout to a very high a number then.

Yeah, that's what I am doing right now.

**Comment by yifeikong** [comment]
I guess you are right, setting None should mean no timeout at all.

**Comment by yifeikong** [comment]
Fixed in `0.5.10b5`.
